ledger-core/v2/insolar/payload/payload.go

```diff
@@ -7,12 +7,17 @@ package payload
 
 import (
 	"encoding/base64"
+	"github.com/insolar/assured-ledger/ledger-core/v2/rms"
 
 	"github.com/gogo/protobuf/proto"
 
 	errors "github.com/insolar/assured-ledger/ledger-core/v2/vanilla/throw"
 )
 
+var RegisterMessageType = rms.RegisterMessageType
```

Comment:
better to redeclare a function - then it will be inlined by the compiler

Reply:
@cyraxred Fixed